src/services/EnvironmentVariables.ts

```diff
@@ -88,4 +90,8 @@ export class EnvironmentVariables {
 		return this.RETURN_JOURNEY_URL;
 	}
 
+	sessionEventsTable(): any {
```

Comment:
This isn't an any. Can we constrain this type to a string?

Reply:
As discussed on slack this is a limitation of the current implementation of the EnvironmentVariables.ts file. Will be changing this to expected types as part of the refactoring work